File: admin/src/app/dtos/supplier-order.dto.ts
import { CartItemDto } from "./cartitem.dto";

export class SupplierOrderDto{
    supplierId:number;
    userId:number;
    totalMoney:number;
    note:string;
    cartItems: CartItemDto[];

  constructor(data:any){
    this.supplierId=data.supplierId;
    this.userId=data.userId;
    this.note=data.note;
    this.totalMoney=data.totalMoney;
    this.cartItems=data.cartItems
  }
}

File: backend/dalathasfarm/src/main/java/com/example/dalathasfarm/dtos/SupplierOrderDto.java
package com.example.dalathasfarm.dtos;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class SupplierOrderDto {
    private int supplierId;
    private int userId;
    private BigDecimal totalMoney;
    private String note;
    private List<CartItemDto> cartItems;
}
